source/includes/steps-repo.yaml

```diff
@@ -133,7 +133,22 @@ content: |
   .. code-block:: sh
 
      git remote add upstream https://github.com/<repoOwner>/<remoteRepoRoot>.git
+---
+title: Remove the redirects file
+ref: remove-redirects
+content: |
+
+  Some of our repositories have a ``redirects`` file that redirects browsers permanently to a new location.
```

Comment:
Suggest: "Some MongoDB docs repositories have a ``redirects file`` that permanently redirects browsers to a new location."